File: 이코테 복습(python, c++, java)/그리디/java/거스름돈.java
package javastudy.이코테.그리디;

public class 거스름돈 {
    public static void main(String[] args) {
        int n = 1260;
        int cnt = 0;
        int[] coinTypes = {500, 100, 50, 10};

        for (int i = 0; i < 4; i++) {
            cnt += n / coinTypes[i];
            n %= coinTypes[i];
        }

        System.out.println(cnt);
    }
}

File: 이코테 복습(python, c++, java)/그리디/c++/거스름돈.cpp
#include<iostream>
using namespace std;

int n = 1260;
int cnt;

int coinTypes[4] = { 500, 100, 50, 10 };

int main(void) {
	for (int i = 0; i < 4; i++) {
		cnt += n / coinTypes[i];
		n %= coinTypes[i];
	}
	cout << cnt << endl;
}
